components/agenda/src/Agenda.svelte

```diff
@@ -4,6 +4,7 @@
   import { tick } from "svelte";
   import { get_current_component, onMount } from "svelte/internal";
   import { spring } from "svelte/motion";
+  import "@commons/components/NError.svelte";
```

Comment:
Not `import "@commons/components/DefaultError.svelte";`?

Reply:
`NError.svelte` is the web component that needs to be imported into every other component except `Composer`, since we've [removed the import from @commons](https://github.com/nylas/components/pull/331/files#diff-a04db6116883df596ab7ac5936e5c55be17e1edb388751e56cc272029972d665L50). Composer has the svelte component `DefaultError.svelte`.